src/platform_impl/windows/window.rs

```diff
@@ -404,12 +400,10 @@ impl Window {
                     // string, so add it
                     display_name.push(0);
 
-                    let mut native_video_mode = video_mode.video_mode.native_video_mode;
-
                     let res = unsafe {
                         winuser::ChangeDisplaySettingsExW(
                             display_name.as_ptr(),
-                            &mut native_video_mode,
+                            &mut *video_mode.video_mode.native_video_mode,
```

Comment:
Not sure about this, we're doing an extra deref here, is that intentional?

Reply:
Because its boxed now.
The deref is to get the inner value.